src/sentry/issues/search.py

```diff
@@ -190,14 +190,46 @@ def _query_params_for_perf(
     return None
 
 
+def _query_params_for_generic(
+    query_partial: SearchQueryPartial,
+    selected_columns: Sequence[Any],
+    aggregations: Sequence[Any],
+    organization_id: int,
+    project_ids: Sequence[int],
+    environments: Optional[Sequence[Environment]],
+    group_ids: Optional[Sequence[int]],
+    filters: Mapping[str, Sequence[int]],
+    conditions: Sequence[Any],
+) -> Optional[SnubaQueryParams]:
+    organization = Organization.objects.filter(id=organization_id).first()
+    if organization and features.has("organizations:issue-platform", organization=organization):
+        if group_ids:
+            filters = {"group_id": sorted(group_ids), **filters}
+
+        params = query_partial(
+            dataset=snuba.Dataset.IssuePlatform,
+            selected_columns=selected_columns,
+            filter_keys=filters,
+            conditions=[],
```

Comment:
do we not want to pass conditions yet?

Reply:
Hmm, I think we do. Looks like my test didn't catch this, I'll update it and fix this as well

Reply:
Tried to fix this, but we're not writing all the fields yet: https://github.com/getsentry/snuba/blob/8a4541f5a0d2bc2e918c25cbfa132ae84ce6a5d9/snuba/datasets/processors/search_issues_processor.py#L128

I'll merge as is without the conditions, and put up a separate pr that'll fix it once we're writing them.